aria-practices.html

```diff
@@ -2358,8 +2358,8 @@ <h3>Slider (Multi-Thumb)</h3>
         with two or more thumbs that each set a value in a group of related values.
         For example, in a product search, a two-thumb slider could be used to enable users to set the minimum and maximum price limits for the search.
         In many two-thumb sliders, the thumbs are not allowed to pass one another, such as when the slider sets the minimum and maximum values for a range.
-        For example, in a price range selector, the maxmimum value of the thumb that sets the lower end of the range is limited by the current value of the thumb that sets the upper end of the range.
-        Conversly the minimum value of the upper end thumb is limited by the current value of the lower end thumb.
+        For example, in a price range selector, the maximum value of the thumb that sets the lower end of the range is limited by the current value of the thumb that sets the upper end of the range.
+        Conversely the minimum value of the upper end thumb is limited by the current value of the lower end thumb.
```

Comment:
Since we are changing this line, please add a comma after conversly.

Reply:
Done